openmdao/docs/usr-guide/tutorials/paraboloid-tutorial.rst

```diff
@@ -257,6 +257,45 @@ The output should look like this:
 
    -15.0
 
+The `IndepVarComp` component is used to define a source for an unconnected
+`param` that we want to use as an independent variable that can be decared as
+a design variabe for a driver. In our case, we want to optimize the
+Paraboloid model, finding values for 'x' and 'y' that minimize the output
+'f_xy.'
+
+Sometimes we just want to run our component once to see the result.
```

Comment:
I don't mind telling them how to run it once. But it should be mentioned that you can also use this to set initial values for the optimization as well.